### PR TITLE
Updated-swap-providers-header

### PR DESCRIPTION
**Pull request by CoderJolly**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by CoderJolly** [comment]
Please look into this PR

**Comment by CoderJolly** [comment]
Test cases have nt been passed, Can some1 pls let know wat is the mistake.....I still want to work on it

**Comment by Gauravano** [comment]
Hi @CoderJolly plz link this PR with the issue, so it can be reviewed. Thanks!

**Comment by CoderJolly** [comment]
I have raised the PR regarding this issue #3033

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.1'
+ruby '2.5.1'
```
Comment (Gauravano) [inline Gemfile]: please exclude `Gemfile` from PR

File: Gemfile.lock [hunk 2]
```diff
@@ -539,7 +539,7 @@ DEPENDENCIES
   will_paginate-bootstrap (>= 1.0.1)
 
 RUBY VERSION
-   ruby 2.4.1p111
```
Comment (Gauravano) [inline Gemfile.lock]: This file need to be removed too

**Comment by Gauravano** [comment]
@CoderJolly please post the screenshot of the new view. Thanks!

**Comment by SidharthBansal** [comment]
@CoderJolly I have resolved the conflicts. Can you please provide us screenshots?

**Comment by CoderJolly** [comment]
give me an hour

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -155,6 +155,23 @@
               <li style="padding:6px;">
                 <%= form_for :user_session, :as => :user_session, :url => { :controller => "user_sessions", :action => "create" }, :html => {:class => "form"} do |f| %>
 
+                <% if Rails.env == "development"%>
```
Comment (SidharthBansal) [inline app/views/layouts/_header.html.erb]: It should be production.

File: Gemfile.lock (hunk 2 above)
Comment (CoderJolly) [reply Gemfile.lock]: Could u please elaborate regarding wat u r referring in gemfile...?

File: Gemfile (hunk 1 above)
Comment (CoderJolly) [reply Gemfile]: ok
Shall i raise the PR again ?

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (CoderJolly) [reply app/views/layouts/_header.html.erb]: Ok....I will do right away

File: Gemfile (hunk 1 above)
Comment (SidharthBansal) [reply Gemfile]: You can either raise a new PR or remove the file from this PR.

**Comment by CoderJolly** [comment]
@SidharthBansal You had requested the screenshots, bt regarding wat...pls tell?

**Comment by SidharthBansal** [comment]
Screenshots for header

**Comment by CoderJolly** [comment]
OK @SidharthBansal

File: Gemfile.lock (hunk 2 above)
Comment (SidharthBansal) [reply Gemfile.lock]: Whole gemfile is not needed

**Pull request closed without merging**